test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -1751,10 +1751,7 @@ def userdel(self, name, del_home=True, force=True, logerr=True,
             if logerr:
```

Comment:
don't capture output of pwd.getpwnam() as codacy complains about unused uinfo variable.